src/Tribe/Views/V2/View.php

```diff
@@ -1705,6 +1717,10 @@ static function () {
 			'show_latest_past'     => $this->should_show_latest_past_events_view(),
 		];
 
+		if ( ! $this->config->get( 'TEC_NO_MEMOIZE_VIEW_VARS' ) ) {
```

Comment:
Same as above - I don't think this is any different than our feature flag we used, for example the CT1 disabled flag. 

My biggest concern is some edge cases that may be found.

Also it would be nice for testing purposes i.e. `define('WP_CACHE', false)`. Most people will ignore this, but if we need a lever in some situations, we have an option here.

I guess this begs the question about feature flags in general?